package.json

```diff
@@ -78,9 +78,11 @@
         "sync-request": "6.0.0",
         "yeoman-environment": "2.3.4",
         "yeoman-generator": "3.2.0",
-        "yo": "3.1.0"
+        "yo": "3.1.0",
+        "progress": "2.0.3"
     },
     "devDependencies": {
+        "aws-sdk": "^2.524.0",
```

Comment:
do we need this ? I don't see any change related to that
if yes, I'd prefer to have a fixed version, to avoid transitive dependencies issues

Reply:
You are right; they are unnecessary; we have a function to install them at runtime. I've added them for testing to save some time. Anyways I've reverted both package.json and package-lock.json :smile: